### PR TITLE
Add option to specify custom kontent.ai url to all commands

### PR DESCRIPTION
**Pull request by JiriLojda**
only for the Management API

### Motivation

Which issue does this fix? Fixes #`issue number`

If no issue exists, what is the fix or new feature? Were there any reasons to fix/implement things that are not obvious?

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

File: src/commands/environment/backupRestore/backup.ts [hunk 1]
```diff
@@ -49,6 +49,10 @@ export const register: RegisterCommand = yargs =>
           alias: "x",
           choices: backupEntityChoices,
           conflicts: "include",
+        })
+        .option("managementApiUrl", {
+          type: "string",
+          describe: "URL of the Kontent.ai Management API.",
```
Comment (IvanKiral) [inline src/commands/environment/backupRestore/backup.ts:55]: Nitpick: what about "Custom URL for Kontent.ai MAPI calls"?

Comment (IvanKiral) [reply src/commands/environment/backupRestore/backup.ts:55]: also migration toolkit commands also has option for baseUrl, maybe add it also there?

**Review by IvanKiral — CHANGES_REQUESTED** [review]

File: src/utils/client.ts [hunk 2]
```diff
@@ -27,32 +30,40 @@ const retryStrategy: IRetryStrategyOptions = {
...
     retryStrategy,
     headers: [{ header: sourceTrackingHeaderName, value: `${packageJson.name};${packageJson.version};${commandName}` }],
     httpService: defaultHttpService,
+    baseUrl: resolveUrlParam(baseUrl, createManagementApiUrl),
   });
 
-export const createClientDelivery = (
-  { environmentId, previewApiKey, usePreviewMode, commandName }: DeliveryParams,
-): DeliveryClient =>
+export const createClientDelivery = (params: DeliveryParams): DeliveryClient =>
```
Comment (pokornyd) [inline src/utils/client.ts:44]: so what's it going to be? destructuring or not? now `createClient` uses destructuring but `createClientDelivery` does not :D

**Review by pokornyd — APPROVED** [review]

File: src/utils/client.ts (hunk 2 above)
Comment (JiriLojda) [reply src/utils/client.ts:44]: I'll unify it 😄

**Pull request merged**